Makefile

```diff
@@ -166,4 +166,12 @@ db-update: ## Run the flyway database migration update (requires maven to be ins
 	@echo "$(P) Run the flyway database migration update"
 	@cd libs/java/dal/db; make db-update;
 
+##############################################################################
+# Kafka Utility Commands
+##############################################################################
+
+reset-consumer-offset: ## Reset the consumer group topic offset.
```

Comment:
Quick and easy way to reset the consumer offset position.  This enables the consumer to pull the same messages again for testing.